docs/QUICK_START_SQL.md

```diff
@@ -0,0 +1,251 @@
+**NOTE: Please read the [python quickstart guide](QUICK_START_PYTHON.md) before reading this section. We will assume
```

Comment:
Given some of the ideas we've discussed recently, I think it may be misleading to call this document QUICK_START_SQL - since it is Python with SQL. Someone only with knowledge of SQL would be unlike to follow this path.